.github/workflows/branch-snapshot.yml

```diff
@@ -28,15 +28,13 @@ jobs:
           go-version: '1.21.4'
       - name: Install Nats Server
         run: |
+          pkill -9 nats-server 2>/dev/null || true
```

Comment:
I originally thought the problem was servers were being left around, and had been meaning to add this anyway.